metagraph/src/annotation/row_diff_builder.cpp

```diff
@@ -282,7 +300,7 @@ rd_succ_bv_type route_at_forks(const graph::DBGSuccinct &graph,
         logger->trace("RowDiff successors will be set to the adjacent nodes with"
                       " the largest number of labels");
 
-        const bit_vector &last = graph.get_boss().get_last();
+        const bit_vector &last = *graph.get_last();
```

Comment:
```suggestion
        const bit_vector &last = graph->get_last();
```
Is there a reason not to do it like this? I think it looks cleaner

Reply:
Yes, it's the equivalent of `*(graph.get_last())` actually, not of `graph->get_last()`.